src/components/common/stateIcon.scss

```diff
@@ -19,6 +19,8 @@
 .resource-state-text-cell {
```

Comment:
Those badges are not under `.resource-state-text-cell`

Reply:
Good catch, fixed.